src/main/java/school/lemon/changerequest/java/introduction/hw2/Task7.java

```diff
@@ -2,6 +2,14 @@
 
 public class Task7 {
     public static int countEvens(int[] arr) {
-        return 0;
+        if (arr == null) {
+            return 0;
+        }
+        int n = 0;
+        for (int i = arr.length - 1; i >= 0; --i)
+            if (arr[i] % 2 == 0 || arr[i] == 0) {
```

Comment:
There is no need for you additional condition `arr[i]==0` cause even when `arr[i]` will be `0` - `0%2 ==0` will return true.